src/utils/coordinates.ts

```diff
@@ -0,0 +1,187 @@
+import { transform } from 'ol/proj'
+
+import { WSG84_PROJECTION } from '../constants'
+import { CoordinatesFormat } from '../fields/CoordinatesInput/constants'
+import { isNumeric } from './isNumeric'
+
+enum CoordinateLatLon {
+  LATITUDE = 'LATITUDE',
+  LONGITUDE = 'LONGITUDE'
+}
+
+/**
+ * Get coordinates in the specified format
+ * @param {number[]} coordinates - Coordinates ([longitude, latitude]) in decimal format.
```

Comment:
Just so you know you don't have to repeat the type in TS JSDoc, you can just write `* @param coordinates My description`. Same thing for the `@returns`.

Reply:
Yep, it was previously a .js file without types